blocks/rich-text/format-toolbar/index.js

```diff
@@ -110,9 +110,22 @@ class FormatToolbar extends Component {
 		this.setState( { isEditingLink: false, isAddingLink: true, newLinkValue: this.props.formats.link.value } );
 	}
 
+	correctLink( link ) {
```

Comment:
Such a function should probably live in https://github.com/WordPress/gutenberg/tree/master/utils

Reply:
Do I need create a new file, right?